cli/src/commands/approve-and-mint-to/index.ts

```diff
@@ -0,0 +1,86 @@
+import { Command, Flags } from "@oclif/core";
+import {
+  CustomLoader,
+  defaultSolanaWalletKeypair,
+  generateSolanaTransactionURL,
+  getSolanaRpcUrl,
+} from "../../utils/utils";
+import { getKeypairFromFile } from "@solana-developers/helpers";
+import { Keypair, PublicKey } from "@solana/web3.js";
+import { approveAndMintTo } from "@lightprotocol/compressed-token";
+import { getTestRpc } from "@lightprotocol/stateless.js";
+
+class ApproveAndMintToCommand extends Command {
+  static summary =
+    "Mint tokens to a compressed account via external mint authority";
+
+  static examples = [
+    "$ light approve-and-mint-to --mint PublicKey --to PublicKey --amount 1000",
+  ];
+
+  static flags = {
+    "mint-authority": Flags.string({
+      description:
+        "Specify the filepath of the mint authority keypair. Defaults to your local solana wallet.",
+      required: false,
+    }),
+    mint: Flags.string({
+      description: "Specify the mint address.",
+      required: true,
+    }),
+    to: Flags.string({
+      description: "Specify the recipient address.",
+      required: true,
+    }),
+    amount: Flags.integer({
+      description: "Amount to mint, in tokens.",
+      required: true,
+    }),
```

Comment:
for required flags wouldn't it make sense to make these arguments?

Reply:
on bucket list for separate pr